lib/shared/types/src/types/apis/sdk/clientSDKAPI.ts

```diff
@@ -111,15 +120,15 @@ export class DVCAPIUser {
      */
     @IsDate()
     @IsOptional()
-    @Transform(({ value }) => value ? new Date(Number(value)) : undefined)
+    @Transform(dateTransform)
```

Comment:
you should just be able to remove the `Number()` call here and pass the value directly into the Date constructor.